### PR TITLE
Update extensions plugin

### PR DESCRIPTION
**Pull request by MsRandom**

### CODE REVIEW

**Review by ThatGravyBoat — APPROVED** [review]
👍💩

File: settings.gradle.kts [hunk 1]
```diff
@@ -12,7 +12,7 @@ pluginManagement {
     }
 }
 
-include("common", "fabric", "forge")
+includeNamed("common", "fabric", "forge")
```
Comment (CodexAdrian) [inline settings.gradle.kts:15]: thats cringe

Comment (CodexAdrian) [reply settings.gradle.kts:15]: - ThatGravityBoat

**Review by CodexAdrian — APPROVED** [review]
lgtm

**Pull request merged**